docs/user/run-shiny-server-operation.md

```diff
@@ -0,0 +1,62 @@
+# How To Run the Riboviz Interactive Data Visualization On Your Data
+
+This page describes how to generate and run an interactive shiny app to visualize and explore your data after running the riboviz workflow. Riboviz 2.0 has an integrated, but optional data visualization tool written in `R` and using `shiny`. The script is located at `riboviz/rscripts/run_shiny_server.R` and takes a single mandatory argument - a YAML file, and one optional argument - a port number for port forwarding. The YAML file is not the one used for a riboviz run, but instead is one called `interactive_viz_config.yaml` which will be located in your output folder.
+
+# Activating the shiny server on a local machine
+
+To use the visualization tool, you can enter the following command at the command line. If you're outside of `riboviz/rscripts`, then you'll need to provide the full path to the `run_shiny_server.R` script.
+```
+Rscript --vanilla run_shiny_server.R path/to/interactive_viz_config.yaml
+```
+
+Once entered, the command will print out the something like the following
+```
+"Listening on http://127.0.0.1:4254"
+```
+
+The IP address provided can simply be pasted into a web browsers URL bar and will direct you to the shiny server which is now hosted from the local machine. 
+
+# Activating the shiny server on a headless machine
+
+If you're running on a headless machine like a server, the shiny server will be hosted on said server. By default, the shiny server is sent to port `4254`. If this port is open, then the same command as above will allow you to access the shiny server. Alternatively, you can specify a port at the command line as in the following example which uses port `1234`.
+```
+Rscript --vanilla run_shiny_server.R path/to/yaml/configuration/file.yaml 1234
+```
+
+You may need to forward that port. *A description of how to forward the port from the command line as opposed to doing it in VSC would be good here...*
```

Comment:
* Incomplete text. Either delete this comment or provide a how-to or a web link.

Reply:
I'll have a short google for this and if I can't find any methods that seem to be quick and easy, I'll remove the comment and create a new ticket for this

Reply:
This issue has been split out as a new ticket: [riboviz/#431](https://github.com/riboviz/riboviz/issues/431), so I'm marking it as resolved here.